static/js/home-app.js

```diff
@@ -4,72 +4,56 @@ $(document).ready(function(){
 });
 
 $(window).load(function () {
-  var codesy = {user:{}}
-
-  Stripe.setPublishableKey('pk_test_NBTdQSXW5HcsEVNgzOZmmZng');
-
-  $('#refreshing').hide();
-
-  if ($("#codesy_user_id").length > 0) {
+    var codesy = {user:{}}
+    Stripe.setPublishableKey('pk_test_NBTdQSXW5HcsEVNgzOZmmZng');
```

Comment:
Here's how I like to get env vars thru settings into static JS ...

`codesy/context_processors.py`:
```
from django.conf import settings


def conf_settings(request):
    return {'settings': settings}
```
`codesy/settings.py`:
```
TEMPLATES = [
    {
        'BACKEND': 'django.template.backends.django.DjangoTemplates',
        'DIRS': [path('templates')],
        'APP_DIRS': True,
        'OPTIONS': {
            'context_processors': [
                'django.template.context_processors.debug',
                'django.template.context_processors.request',
                'django.contrib.auth.context_processors.auth',
                'django.contrib.messages.context_processors.messages',

                'codesy.context_processors.conf_settings'
            ],
        },
    },
]
```
`base.html`:
```
<html id="codesy-html" data-stripe_key="{{ settings.STRIPE_KEY }}">
```
`static/js/home-app.js`:
```
var stripe_key = document.getElementById("codesy-html").dataset.stripe_key;
Stripe.setPublishableKey(stripe_key);
```